config/sync/xmlsitemap.xmlsitemap.NXhscRe0440PFpI5dSznEVgmauL25KojD7u4e9aZwOM.yml

```diff
@@ -0,0 +1,7 @@
+uuid: 483ccfbb-31f8-4b1a-896d-c4e55e23902a
+langcode: en
+status: true
+dependencies: {  }
+id: NXhscRe0440PFpI5dSznEVgmauL25KojD7u4e9aZwOM
+label: 'http://digitalgov.lndo.site'
```

Comment:
This should just be Digital.gov, it's only a label and what will be shown on all envs.
/admin/config/search/xmlsitemap

Reply:
updated label to be `Digital.gov`